store/acistore/store.go

```diff
@@ -97,39 +96,37 @@ func (e *StoreRemovalError) Error() string {
 	return s
 }
 
-// Store encapsulates a content-addressable-storage for storing ACIs on disk.
-type Store struct {
-	dir       string
-	stores    []*diskv.Diskv
-	db        *db.DB
-	treestore *TreeStore
+// ACIStore encapsulates a content-addressable-storage for storing ACIs on disk.
+type ACIStore struct {
```

Comment:
this stutters a lot with the package name, can we call this simply `Store`? this would then become `acistore.Store`, which still stutters, but less ;-)